### PR TITLE
Seven seg display

### PR DESCRIPTION
**Pull request by TDW89**
prints a seven seg display.

### CODE REVIEW

File: doc/lib_seven_seg.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+###seven_seg
+
+args:
+  * Input  ``Number``. This should be between 0 and 9 (or `"b"` to blank the display and `"-"` for minus).
+  * Column ``Number``. The left most column of the display (it is 3 columns wide).
+  * Line  ``Number``. The top line of the display (it is 3 lines hight).
+  
+description:
+  * Prints a seven segmant display on the terminal showing the input value at the location specified.
```
Comment (space-is-hard) [inline doc/lib_seven_seg.md]: *segment :)

File: doc/lib_seven_seg.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+###seven_seg
```
Comment (space-is-hard) [inline doc/lib_seven_seg.md]: Needs licensing

File: examples/example_lib_seven_seg.ks [hunk 3]
```diff
@@ -0,0 +1,32 @@
+run lib_seven_seg.
```
Comment (space-is-hard) [inline examples/example_lib_seven_seg.ks]: Needs licensing

File: doc/lib_seven_seg.md (hunk 2 above)
Comment (TDW89) [inline doc/lib_seven_seg.md]: @space-is-hard do we want licensing on the docs as well?

File: examples/example_lib_seven_seg.ks (hunk 3 above)
Comment (TDW89) [inline examples/example_lib_seven_seg.ks]: added.

File: doc/lib_seven_seg.md (hunk 2 above)
Comment (space-is-hard) [inline doc/lib_seven_seg.md]: Yeah, we did write them so I assume we need it. It's been the practice so far.

**Pull request merged**